packages/next-swc/Cargo.toml

```diff
@@ -46,36 +46,37 @@ swc_emotion = { version = "0.29.10" }
+turbo-tasks-env = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbo-tasks-fetch = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbo-tasks-fs = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbo-tasks-hash = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbo-tasks-macros = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbo-tasks-macros-shared = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbo-tasks-memory = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbo-tasks-testing = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbo-updater = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-cli-utils = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-core = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-create-test-app = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-css = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-dev = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-dev-server = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-ecmascript = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-env = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-json = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-mdx = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-node = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-static = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-swc-utils = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-test-utils = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
+turbopack-tests = { git = "https://github.com/vercel/turbo.git", rev = "0343d350deda806249800f8561618a5532bdd960" }
```

Comment:
I'll update this to the turbopack release that includes https://github.com/vercel/turbo/pull/4141